.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 *.iml
 composer.phar
+app-prod.yaml
```

Comment:
I like `app.yaml.template` (or `app.yaml.dist` is common in PHP), but what secrets do you expect to have in `app.yaml`?  DB passwords?